### PR TITLE
[usage] Proto changes for GetCostCenter RPC

### PR DESCRIPTION
**Pull request by laushinka**
## Description
Proto changes for the GetCostCenter RPC

## Related Issue(s)

Relates #12264

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: components/usage-api/usage/v1/usage.proto [hunk 1]
```diff
@@ -66,3 +69,16 @@ message ReconcileUsageResponse {
...
+    string attribution_id = 1;
+}
+
+message GetCostCenterResponse {
+    CostCenter cost_center = 1;
+}
+
+message CostCenter {
+    string attribution_id = 1;
+    int32 spending_limit = 2;
```
Comment (easyCZ) [inline components/usage-api/usage/v1/usage.proto:83]: What's the precision stored in the DB? Is `int32` enough? I'd probably err on the side of caution here and go for int64

**Review by easyCZ — APPROVED** [review]
/hold for Q around precision

File: components/usage-api/usage/v1/usage.proto (hunk 1 above)
Comment (laushinka) [reply components/usage-api/usage/v1/usage.proto:83]: It's `int(11)` which I understand to be 32 bits?

Comment (laushinka) [reply components/usage-api/usage/v1/usage.proto:83]: I worry if we set it to `int64` we might truncate something during the conversion? 

I will unhold for now to unblock, but we can discuss again what we should store.

**Pull request merged**